internal/metrics/middleware.go

```diff
@@ -102,61 +108,44 @@ func HTTPMetricsHandler(service string) func(next http.Handler) http.Handler {
 
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			startTime := time.Now()
-			m := responsewriter.NewWrapResponseWriter(w, 1)
-
-			next.ServeHTTP(m, r)
-
 			ctx, tagErr := tag.New(
-				context.Background(),
```

Comment:
Good catch! I missed this.

Reply:
It was only being used for record() within the handler previously.